Plugins/Feedback/NWScript/nwnx_feedback.nss

```diff
@@ -44,15 +74,92 @@ int NWNX_Feedback_GetFeedbackMessageHidden(int nMessage, object oPC = OBJECT_INV
 
 void NWNX_Feedback_SetFeedbackMessageHidden(int nMessage, int nState, object oPC = OBJECT_INVALID)
 {
-    string sFunc = "SetFeedbackMessageHidden";
+    string sFunc = "SetMessageHidden";
+    int nMessageType = 0;
 
     NWNX_PushArgumentInt(NWNX_Feedback, sFunc, nState);
     NWNX_PushArgumentInt(NWNX_Feedback, sFunc, nMessage);
+    NWNX_PushArgumentInt(NWNX_Feedback, sFunc, nMessageType);
     NWNX_PushArgumentObject(NWNX_Feedback, sFunc, oPC);
     NWNX_CallFunction(NWNX_Feedback, sFunc);
 }
 
-/* Feedback Messages
+int NWNX_Feedback_GetCombatLogMessageHidden(int nMessage, object oPC = OBJECT_INVALID)
+{
+    string sFunc = "GetMessageHidden";
+    int nMessageType = 1;
+
+    NWNX_PushArgumentInt(NWNX_Feedback, sFunc, nMessage);
+    NWNX_PushArgumentInt(NWNX_Feedback, sFunc, nMessageType);
+    NWNX_PushArgumentObject(NWNX_Feedback, sFunc, oPC);
+    NWNX_CallFunction(NWNX_Feedback, sFunc);
+
+    return NWNX_GetReturnValueInt(NWNX_Feedback, sFunc);
+}
+
+void NWNX_Feedback_SetCombatLogMessageHidden(int nMessage, int nState, object oPC = OBJECT_INVALID)
+{
+    string sFunc = "SetMessageHidden";
+    int nMessageType = 1;
+
+    NWNX_PushArgumentInt(NWNX_Feedback, sFunc, nState);
+    NWNX_PushArgumentInt(NWNX_Feedback, sFunc, nMessage);
+    NWNX_PushArgumentInt(NWNX_Feedback, sFunc, nMessageType);
+    NWNX_PushArgumentObject(NWNX_Feedback, sFunc, oPC);
+    NWNX_CallFunction(NWNX_Feedback, sFunc);
+}
+
+/*
+// CombatLog Messages
+// For use with NWNX_Feedback_SetCombatLogMessageHidden() and
+//              NWNX_Feedback_GetCombatLogMessageHidden()
+
+const int COMBATLOG_SIMPLE_ADJECTIVE    = 1;
```

Comment:
Prefix these with `NWNX_FEEDBACK_`, don't collisions when someone imports this into their module.

Reply:
Done, prefixed the feedback ones with NWNX_ too for good measure